packages/blocks/src/routes/following-marker.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
```

Comment:
This example is basically how we want to use MapLibre in the movement sensor debug card.